grafana/types.json

```diff
@@ -1215,7 +1215,7 @@
       "fieldConfig": {
         "defaults": {
           "class": "fieldConfig_defaults",
-          "unit": "si:writes/s"
+          "unit": "si:writes"
```

Comment:
What about "reads_panel", @amnonh

Reply:
I looked at both - hence the question. ;)

Reply:
@vladzcloudius  it seems to work on my side, did you try and it didn't? if so maybe there's a commit that is missing

Reply:
I did not try it. I was just looking at the code and I saw that you are using both `reads_panel` and `writes_panel` in the second patch but only update `writes_panel` in the first. 

Hence my question.
But if you tell me that you checked it and it works - you can ignore it.